com.unity.render-pipelines.universal/Runtime/UniversalAdditionalCameraData.cs

```diff
@@ -222,11 +222,24 @@ public List<Camera> cameraStack
                     Debug.LogWarning(string.Format("{0}: This camera has a ScriptableRenderer that doesn't support camera stacking. Camera stack is null.", camera.name));
                     return null;
                 }
-
+                updateCameraStack();
                 return m_Cameras;
             }
         }
 
+        public void updateCameraStack()
```

Comment:
Make this Private and make it Uppercase U
UpdateCameraStack()

Reply:
UpdateCameraStack needs to be public such that it can be accessed when calling the UpdateCameras in UniversalRenderPipelineCameraEditor.cs because it accesses the value m_cameras instead of the getter.